### PR TITLE
Update ja-JP using v2 SDK

### PR DESCRIPTION
**Pull request by Kazushige-Yoshida-A**
Updated ja-JP branch based on the newest en-US branch using v2 SDK

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +15,23 @@ Alexaはこれらの問いかけに対して次のように答えます。
 *  「知ってましたか？ラルフ・カイナーはメジャーリーグデビュー後、七年連続で本塁打王になったただ一人の選手です。」
 *  「知ってましたか？タイタニック号は一日に600トンの石炭を消費しました。176人の乗務員がシャベルでボイラーに石炭を投入し、毎日100トンの灰を海に放出していました。」
 
-<!--このスキルの動作例を見たい場合は、[Alexaスキルストア](http://amazon.com/skills)から[Gloucester Facts](https://www.amazon.com/Robert-McCauley-Gloucester-Facts/dp/B01I5MOIA2/)を有効化して使ってみてください。-->
-
+**開始** するには次のボタンをクリックしてください。
```
Comment (SleepyDeveloper) [inline README.md]: The image should be on it's own line.

**Review by SleepyDeveloper — COMMENTED** [review]
Looks good other than formatting issue on README.md where the "始めましょう" image is not appearing on its own line.

File: README.md (hunk 1 above)
Comment (Kazushige-Yoshida-A) [reply README.md]: I tried to fix it. Please check.

**Comment by SleepyDeveloper** [comment]
LGTM 👍

**Pull request merged**